test/wasi/c/sock.c

```diff
@@ -0,0 +1,17 @@
+#include <sys/socket.h>
+#include <stddef.h>
+#include <errno.h>
+#include <assert.h>
+#include <stdio.h>
+
+int main(void) {
+
```

Comment:
Since this file will need to be updated when there is a full implementation of `sock_accept()`, could you leave a `TODO` comment.

Nit: blank line.